src/components/products/Products.component.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+interface CategoryProps {
+    id: number;
+    name: string;
+    image: string;
+    creationAt: string;
+    updatedAt: string;
+}
+
+type ExcludedCategoryProps = Exclude<CategoryProps, 'image' | 'creationAt' | 'updatedAt'>;
+
+interface ProductProps {
+    id: number;
+    title: string;
+    price: number;
+    description: string;
+    images: string[];
+    creationAt: string;
+    updatedAt: string;
+    category: ExcludedCategoryProps;
+}
+
+type ExcludedProductProps = Exclude<ProductProps, 'creationAt' | 'updatedAt'>;
+
+function formatPrice(price: number): string {
+    return price.toLocaleString('uk-UA');
+}
+
+export const ProductsComponent: React.FC<ExcludedProductProps> = ({ id, title, price, description, images, category }) => (
```

Comment:
I just left it as an example of usage because I didn't have a full understanding of future tasks regarding categories and products. In the real task, I planned to remove it and execute according to the assignment. In this file, I left the interface with a possible way of usage